### ISSUE TITLE
Compare package download counts

### ISSUE THREAD

**Issue by rand0mdev** [composer/packagist#1142]
Hi,
It could be very interesting to add a "**Compare package download counts over time**" like [npmtrends.com](https://www.npmtrends.com/)

I can help to contribute on this feature if you approve it.

**Comment by 94noni** [comment]
For fun and to test things I've POC https://github.com/94noni/packagist-stats

**Comment by bechir** [comment]
@94noni are you planning to add your POC on this repo?

**Comment by 94noni** [comment]
@bechir hey, for now I have no plans and I think it should be asked first to @Seldaek if such feature can be useful on the website http://packagist.org
if so, we could take inspiration on this old poc to implement a first version :)

**Comment by Seldaek** [comment]
Yeah I am not against having this feature but it is not something I'd prioritize for my own time for now..

**Comment by 94noni** [comment]
Repassing by this issue
Perhaps it can be a symfony ux feature someone wants to contribute to via https://ux.symfony.com/demos/live-component/chartjs

Replacing old chart while adding this « comparison » chart
